Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift

```diff
@@ -68,6 +68,8 @@ final class SubscriptionsViewController: BaseViewController {
         updateData()
 
         SocketManager.addConnectionHandler(token: socketHandlerToken, handler: self)
+
+        tableView.reloadData()
```

Comment:
Is this call really required? I think it's not.

Reply:
You're right 👍